api/report_server.thrift

```diff
@@ -134,6 +152,13 @@ struct ReportDataTypeCount{
 }
 typedef list<ReportDataTypeCount> ReportDataTypeCountList
 
+struct Counter{
```

Comment:
This struct is unused, the `getFoobarCounts` all return simply `map<K, i64>`.

Reply:
I've removed it.